Using selectors inside collector (#380)
giantswarm/app-operator: CHANGELOG.md

- Delete helm-2to3-migration job after migration is finished.
- Sending metrics with app CR's version in the spec.

> - Only send metrics from app CRs that reconciled from the controller. 

Reviewer: ```suggestion
- Only emit metrics for app CRs reconciled by this instance of the operator.
```
